File: Python/emirp.py
def is_prime(n):
    if n <= 1:
        return False
    for i in range(2, int(n**0.5) + 1):
        if n % i == 0:
            return False
    return True

def is_emirp(n):
    if not is_prime(n):
        return False
    reversed_n = int(str(n)[::-1])
    if n == reversed_n:
        return False
    return is_prime(reversed_n)

# Example usage
number = int(input("Enter a number: "))
if is_emirp(number):
    print(f"{number} is an Emirp number.")
else:
    print(f"{number} is not an Emirp number.")

File: Python/emirp.js
let x=121
let res=isEmirp(x)?"Emirp":"Not Emirp"
console.log(res)

function isEmirp(x){
    if(isPrime(x)){
        let rev=reverse(x)
        if(isPrime(rev)){
            if(isNotPalindrome(x)){
                return true
            }
        }
    }
    return false
}
function isPrime(n){
    for(let i=2;i<=Math.sqrt(n);i++){
        if(n%i==0){
            return false
        }
    }
    return true
}
function reverse(n){
    let rev=0
    while(n>0){
        rev=rev*10+(n%10)
        n=Math.floor(n/10)
    }
    return rev 
}

function isNotPalindrome(n){
    if(n==reverse(n)){
        return false
    }
    return true
}
